MonoGame.Framework/Input/Mouse.Windows.cs

```diff
@@ -31,7 +36,7 @@ private static void PlatformSetPosition(int x, int y)
             PrimaryWindow.MouseState.X = x;
             PrimaryWindow.MouseState.Y = y;
             
-            var pt = Window.PointToScreen(new System.Drawing.Point(x, y));
+            var pt = _window.PointToScreen(new System.Drawing.Point(x, y));
```

Comment:
Because `_window` is a static variable, this would cause problems with creating multiple instances of `Game`.